com.yutovo.yutovo_desktop.yaml

```
@@ -0,0 +1,27 @@
+app-id: com.yutovo.yutovo_desktop
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+command: run.sh
+finish-args:
+  - --socket=x11
+  - --env=LD_LIBRARY_PATH=/app/bin:$LD_LIBRARY_PATH
+modules:
+  - name: yutovo_desktop
+    buildsystem: simple
+    build-commands:
+      - cp -r bin/ /app/bin/
+      - mkdir -p /app/share/icons/hicolor/scalable/apps
+      - mkdir /app/share/metainfo
+      - mkdir /app/share/applications
+      - install -Dm644 /app/bin/com.yutovo.yutovo_desktop.png /app/share/icons/hicolor/scalable/apps/
+      - install -Dm644 /app/bin/com.yutovo.yutovo_desktop.desktop /app/share/applications/
+      - install -Dm644 /app/bin/com.yutovo.yutovo_desktop.metainfo.xml /app/share/metainfo/
```

```suggestion
      - install -Dm644 /app/bin/com.yutovo.yutovo_desktop.png -t /app/share/icons/hicolor/scalable/apps/
      - install -Dm644 /app/bin/com.yutovo.yutovo_desktop.desktop -t /app/share/applications/
      - install -Dm644 /app/bin/com.yutovo.yutovo_desktop.metainfo.xml -t /app/share/metainfo/
```

And https://github.com/flathub/flathub/pull/6603/files#r2133384769 still applies regardless.